package.json

```diff
@@ -73,8 +78,8 @@
   "main": "lib/js/index.js",
   "module": "lib/es/index.js",
   "peerDependencies": {
-    "react": ">=15",
-    "react-dom": ">=15"
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2"
```

Comment:
@Rendez maybe we can use `>=` range to make sure next major release won't produce warnings on install?

Reply:
A major release could bring breaking changes, I feel it's safer this way.